### ISSUE TITLE
1с falls when testing and repair database

### ISSUE THREAD

**Issue by warlomak** [sam88651/UsbHasp#2]
The hasp drivers do not affect, the tests were on a clean ubuntu, key was forwarded using usb redirector to windows machine, hasp drivers installed correctly. no error messages in logs.
but server key work fine.
![image](https://user-images.githubusercontent.com/37336181/52534007-936b4500-2d5d-11e9-85c3-510e09836e62.png)
![image](https://user-images.githubusercontent.com/37336181/52534014-a1b96100-2d5d-11e9-9d1a-869a00cef2a4.png)
 lsusb
Bus 002 Device 003: ID 0529:0001 Aladdin Knowledge Systems HASP copy protection dongle
Bus 002 Device 002: ID 0529:0001 Aladdin Knowledge Systems HASP copy protection dongle
Bus 002 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 001 Device 002: ID 80ee:0021 VirtualBox USB Tablet
Bus 001 Device 001: ID 1d6b:0001 Linux Foundation 1.1 root hub

# uname -a
Linux hsrv 4.15.0-43-generic #46-Ubuntu SMP Thu Dec 6 14:45:28 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

# dmesg
[   13.631418] tusbd: usbd_stub_reset: trying to reset device vid 80ee pid 0021
[   13.744693] usb 2-2: new full-speed USB device number 3 using usb_vhci_hcd
[   13.876365] usb 1-1: reset full-speed USB device number 2 using ohci-pci
[   13.892850] usb 2-2: New USB device found, idVendor=0529, idProduct=0001
[   13.892853] usb 2-2: New USB device strings: Mfr=1, Product=2, SerialNumber=0
[   13.892856] usb 2-2: Manufacturer: HASP HL 3.25
[   14.187392] tusbd: usbd_stub_reset: usb_reset_device returned 0

so the problem with UDP and hasplm can be eliminated.

**Comment by sam88651** [comment]
I'm aware of this behaviour, As I stated this is a research project. Nor Sentinel, nor 1C support old HASP4 keys. They we declared as EOL, as well as API for them. So there is no wonder it might not work. I suspect that the trouble lays somewhere in algorithms 1C checks legacy HASP licenses. Recently 1C started checking licensing even on configurations level.
I may only suggest to use real keys for heavy DB operations/updates and this project may work on daily operations.
BTW. It's 1C client who loses key, server doesn't even under a high load.
